assets/admin/editor-wizard/steps/course-details-step.js

```diff
@@ -16,71 +20,79 @@ import LimitedTextControl from '../../../blocks/editor-components/limited-text-c
  * @param {Function} props.setData
  */
 const CourseDetailsStep = ( { data: wizardData, setData: setWizardData } ) => {
-	// TODO Replace this sample implementation.
-	const updateNewCourseTitle = ( value ) => {
-		setWizardData( { ...wizardData, newCourseTitle: value } );
+	usePostTitle( wizardData.courseTitle );
+
+	const updateCourseTitle = ( value ) => {
+		setWizardData( { ...wizardData, courseTitle: value } );
 	};
-	const updateNewCourseDescription = ( value ) => {
+
+	const updateCourseDescription = ( value ) => {
 		setWizardData( {
 			...wizardData,
-			newCourseDescription: value,
+			courseDescription: value,
 		} );
 	};
+
 	return (
 		<div className="sensei-editor-wizard-modal__columns">
 			<div className="sensei-editor-wizard-modal__content">
 				<h1>Course Details Step</h1>
-				<div>
+				<div className={ 'sensei-editor-wizard-step__description' }>
```

Comment:
Hey, could you please use "raw" strings attribute value (not sure how to call them, to be honest) here. Like this:

```suggestion
				<div className="sensei-editor-wizard-step__description">
```